py/tests/plane_move.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/python3
+
+import pykms
+import random
+import time
+import sys
+import select
+import selectors
+
+if len(sys.argv) not in (1, 3):
+    print('Without parameters: list of available connectors')
+    print('{} [name of connector0] [name of connector1]\n'.format(sys.argv[0]))
+    sys.exit(-1)
+
+card = pykms.Card()
+
+if not card.has_atomic:
+    print('Atomic mode settings is not supported :(')
```

Comment:
"modesetting", and please, no smileys in the code =)

Reply:
Changed to 'Atomic modesetting is not supported'